lib/clients/http_client/types.ts

```diff
@@ -6,11 +6,13 @@ export interface HeaderParams {
   [key: string]: string | number | string[];
 }
 
+/* eslint-disable @shopify/typescript/prefer-pascal-case-enums */
 export enum DataType {
-  JSON = 'application/json', // eslint-disable-line @shopify/typescript/prefer-pascal-case-enums
-  GraphQL = 'application/graphql', // eslint-disable-line @shopify/typescript/prefer-pascal-case-enums
-  URLEncoded = 'application/x-www-form-urlencoded', // eslint-disable-line @shopify/typescript/prefer-pascal-case-enums
+  JSON = 'application/json',
```

Comment:
Are they no Javascript constant we could use instead of defining them ourselves? I know it would not add much value but I am just cautious redefining things manually as it always open the door to a future mistake

Reply:
Unfortunately no, we use `@shopify/network` for those shared constants. It has pre-defined values for the header names (which we use here), but not for the MIME types :(

Reply:
Ah ok I understand worth raising an issue there or even a PR no ? Or do you think thats not the right place for this sort of constant?